src/main-process/atom-application.coffee

```diff
@@ -322,6 +324,8 @@ class AtomApplication
 
     @disposable.add ipcHelpers.on ipcMain, 'did-cancel-window-unload', =>
       @quitting = false
+      for window in @windows
+        window.didCancelWindowUnload()
```

Comment:
Might not be more readable, and certainly not important, but these two lines could be compacted into
```coffee
window.didCancelWindowUnload() for window in @windows
```